dvc/commands/data_sync.py

```diff
@@ -14,7 +14,14 @@ def log_summary(self, stats):
         from dvc.utils.humanize import get_summary
 
         default_msg = "Everything is up to date."
-        ui.write(get_summary(stats.items()) or default_msg)
+
+        summary = get_summary(stats.items())
+        if not summary:
```

Comment:
Do we even need this condition?

Reply:
You are right, there is no real need for this. Was trying to narrow down the cases when we would show this, but that's not really needed. Will adjust. Thanks.